lib/Serialization/ModuleFormat.h

```diff
@@ -2117,6 +2117,12 @@ namespace decls_block {
     BCFixed<2>  // the builtin conformance kind
   >;
 
+  using AbstractConformanceLayout = BCRecordLayout<
```

Comment:
You forgot to bump the module format

Reply:
My local tree was out of date and I bumped it in *exactly* the same way you did in https://github.com/swiftlang/swift/pull/80107 :)